### PR TITLE
Added Freshers Salary Prediction Web App

### PR DESCRIPTION
**Pull request by ritikkumar55**
## Related Issue 

- Info about the related issue 

- [x] LGM-SOC'21 Participant 
- [ ] Contributor

Closes: #159 

### Describe the add-ons or changes you've made

Give a clear description of what have you added or modifications made
 have added a machine learning based web app using streamlit main goal of this project to is to predict the freshers salary on the basis of entrance test score, subjects, mba percentages and so on. I have performed Exploratory Data Analysis , data visualization  to gain useful insights from the data. Performed label encoding on categorical data and performed various regresssion models to predict the salary. Compared various regression  model like linear regression , xgboost regression , lasso regression, random forest regression etc. and do cross validation and hyperparameter tuning 
## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how it has been tested
Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (Only for Front End and UI/UX Designers)

 Original           | Updated
 :--------------------: |:--------------------:
 Original Screenshot | Updated Screenshot |
[](https://postimg.cc/z3fQpTVr) [](https://postimg.cc/R3V19LrC) [](https://postimg.cc/Z92Wg84M) [](https://postimg.cc/sGygdK8w)

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
Fab visualizations and very well presented. Amazing work. Great going. Keep rocking. Your PR is approved. Congo. 👌🎊@ritikkumar55
